### ISSUE TITLE
[Feature Request]: Include Winget Packages in Release Process

### ISSUE THREAD

**Issue by andylizf** [hovancik/stretchly#1338]
### Version

- [X] I'm using version 1.14.1

### Existing requests

- [X] I've checked Existing requests

### Advanced Preferences

- [X] I've checked Advanced Preferences

### What operating system are you using?

Windows

### Operating System Version

Windows 11 version 22H2

### Problem description

I would like to request the addition of Winget packages to the release process of the Strectchly repository.

### Possible solution

I propose that with each new release of Strectchly, a corresponding Winget package is also created and made available. This would greatly simplify the installation process for users who prefer using Winget as their package manager.

### Alternative solutions

Instead of creating and maintaining Winget packages alongside the regular releases, an alternative solution could be to provide a script or tool that automates the process of generating Winget manifests for Strectchly.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by hovancik** [comment]
Is it possible to create automated wingets? I am not a windows user so I have no idea. From what I can see, somebody somewhere makes Stretchly for winget: https://github.com/microsoft/winget-pkgs/tree/master/manifests/s/Stretchly/Stretchly

**Comment by andylizf** [comment]
You are correct that an existing Winget package for Stretchly is available in winget-pkgs repository. However, the version provided in the manifest is not the latest one.

To ensure that the Winget package is up to date with each release, it would be beneficial to simultaneously update the winget-pkgs repository when releasing new versions of Stretchly. Additionally, setting up a GitHub Action to automate the creation and update of the winget-pkgs repository would further streamline the process.

By doing so, Windows users will have the convenience of quickly installing and updating Stretchly using their preferred package manager.

**Comment by hovancik** [comment]
I would gladly accept a PR or example PR of another project :)

**Comment by andylizf** [comment]
Probably related to [winget-pkgs#1515](https://github.com/microsoft/winget-pkgs/issues/1515) or [actions/winget-releaser](https://github.com/marketplace/actions/winget-releaser)

**Comment by generrosity** [comment]
Aaaah, this makes more sense to why the freshly installed app is complaining about updates available 😅

+1 to using automation, especially if that second on is 'inbuilt' to github.

**Comment by hovancik** [comment]
We can do this once https://github.com/russellbanks/Komac/issues/967#issuecomment-2576528439 is released
